takin_motors/src/motors_control.cpp

```diff
@@ -96,7 +96,9 @@ void setUpMotors(ros::NodeHandle &nh) {
         left_track.push_back(std::make_shared<TalonSRX>(FL));
     }
     if (nh.getParam("/motors_control/rear_left", RL)) {
-        left_track.push_back(std::make_shared<TalonSRX>(RL));
+        std::shared_ptr<TalonSRX> motor = std::make_shared<TalonSRX>(RL);
+        motor->SetInverted(true);
```

Comment:
```suggestion
        //motor->SetInverted(true);
```
Le bug de la drive 61 semble venir d'ici. @Ludoenso

Reply:
C'est une vielle PR. De plus, c'était un bug qu'on avait avant, la drive est inverse physiquement pcq c'est pas une configuration software. Cette ligne là est justement là pour régler le fait qu'elle était inversé.